R/databricks.R

```diff
@@ -47,10 +47,10 @@ setClass("DatabricksOdbcDriver", contains = "OdbcDriver")
 #' @export
 setMethod(
   "dbConnect", "DatabricksOdbcDriver",
-  function(drv, http_path, workspace = Sys.getenv("DATABRICKS_HOST"),
+  function(drv, HTTPPath, workspace = Sys.getenv("DATABRICKS_HOST"),
```

Comment:
I renamed the `http_path` parameter to `HTTPPath` throughout so that it matches the parameter name that would be required when connecting with `odbc::odbc()`.

Hopefully this will prevent confusion when switching between these two methods.

Reply:
Changing this broke CI, so I need to address that and add the commit back in.

Reply:
@atheriel Do you have any concerns about changing this to `HTTPPath`?

Reply:
Or `httpPath` if we want to standardise on lowerCamelCase?